packages/asset-swapper/test/market_operation_utils_test.ts

```diff
@@ -1166,28 +1166,6 @@ describe('MarketOperationUtils tests', () => {
                 expect(orderSources.sort()).to.deep.eq(expectedSources.sort());
             });
 
-            it('fallback orders use different sources', async () => {
```

Comment:
We don't use a different source. We can pad more to the currently used source or add a different source. Tests in `path_test.ts`